src/api/emoji/getEmojiNameToCodePoint.js

```diff
@@ -0,0 +1,6 @@
+/* @flow strict-local */
+import type { Auth } from '../apiTypes';
+import { apiGet } from '../apiFetch';
+
+export default async (auth: Auth): Promise<string[]> =>
+  apiGet(auth, 'static/generated/emoji/name_to_codepoint.json', res => res, undefined, false, true);
```

Comment:
Note that while this file still exists on some Zulip servers as a leftover from previous deployments, it’s no longer being updated as of zulip/zulip#13831. There’s now a static/generated/emoji/emoji_codes.json instead. (We don’t guarantee this as part of the public API though.)